os/arch/arm/src/imxrt/imxrt_usbhost.c

```diff
@@ -194,8 +192,25 @@ int imxrt_usbhost_initialize(void)
 
 	/* Then get an instance of the USB EHCI interface. */
 
-	g_ehciconn = imxrt_ehci_initialize(0);
-	if (!g_ehciconn) {
+#ifdef CONFIG_USBHOST_HUB
+	/* Initialize USB hub class support */
+	if (usbhost_hub_initialize() != OK) {
```

Comment:
you already defined ret, so line 197 & 202 & 213 can be aligned with line 207~210(use ret & add ret to debug message)

Reply:
Corrected

Reply:
Corrected!